connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -518,6 +518,10 @@ class ClientE2ETestSuite extends RemoteSparkSession {
     assert(!spark.conf.isModifiable("spark.sql.globalTempDatabase"))
     intercept[Exception](spark.conf.set("spark.sql.globalTempDatabase", "/dev/null"))
   }
+
+  test("SparkVersion") {
+    assert(!spark.version.isEmpty)
```

Comment:
Does this fail on the `master` branch previously?

Reply:
This in the past load the version directly from core, which was ok given that Spark Connect is depending on the core already. This change is to make that the API does not directly load from core but through the server, which is a long term solution.